From f0b8a736d010fde2d6dc4d284be2d0bb8b32400f Mon Sep 17 00:00:00 2001
From: Sunil Goutham <sgoutham@marvell.com>
Date: Mon, 27 Aug 2018 14:49:42 +0530
Subject: [PATCH 0162/1051] net: octeontx2: Fix VF driver module description

Changed the description dumped on console upon module
init to be consistent with other drivers.

octeontx2-cgx: Marvell OcteonTX2 CGX/MAC Driver
octeontx2-af: Marvell OcteonTX2 RVU Admin Function Driver
octeontx2-nicpf: Marvell OcteonTX2 NIC Physical Function Driver
octeontx2-nicvf: Marvell OcteonTX2 NIC Virtual Function Driver

Signed-off-by: Sunil Goutham <sgoutham@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/otx2_vf.c | 15 ++++++++++++++-
 1 file changed, 14 insertions(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/otx2_vf.c b/drivers/net/ethernet/marvell/octeontx2/otx2_vf.c
index d75c71a522a1..620b92fa743a 100644
--- a/drivers/net/ethernet/marvell/octeontx2/otx2_vf.c
+++ b/drivers/net/ethernet/marvell/octeontx2/otx2_vf.c
@@ -501,4 +501,17 @@ static struct pci_driver otx2vf_driver = {
 	.remove = otx2vf_remove,
 };
 
-module_pci_driver(otx2vf_driver);
+static int __init otx2vf_init_module(void)
+{
+	pr_info("%s: %s\n", DRV_NAME, DRV_STRING);
+
+	return pci_register_driver(&otx2vf_driver);
+}
+
+static void __exit otx2vf_cleanup_module(void)
+{
+	pci_unregister_driver(&otx2vf_driver);
+}
+
+module_init(otx2vf_init_module);
+module_exit(otx2vf_cleanup_module);
-- 
2.17.1

